packages/image_picker/image_picker_platform_interface/lib/src/platform_interface/image_picker_platform.dart

```diff
@@ -141,11 +141,11 @@ abstract class ImagePickerPlatform extends PlatformInterface {
   ///
   /// In Android, the MainActivity can be destroyed for various reasons. If that happens, the result will be lost
   /// in this call. You can then call [retrieveLostData] when your app relaunches to retrieve the lost data.
-  Future<PickedFile> pickImage({
-    @required ImageSource source,
-    double maxWidth,
-    double maxHeight,
-    int imageQuality,
+  Future<PickedFile?> pickImage({
```

Comment:
Same comment about `Future<PickedFile?>`.  We would need to either:
1. (Preferred) Remove the nullability.
2. Explain when the Future completes as `null`, and what to do in those cases.

Reply:
done. Added docs to explain the null value.